app/models/student_validator.rb

```diff
@@ -147,6 +153,10 @@ def has_insurance?
     insurance == EnumConstants::BOOLEANS[0]
   end
 
+  def media_consent?
+    media_participation == EnumConstants::BOOLEANS[0]
```

Comment:
We should check `current_page == 6` here as well! This similar logic should apply to all these special conditional validations.